internal/providers/qemu/qemu.go

```diff
@@ -38,12 +41,22 @@ func FetchConfig(f *resource.Fetcher) (types.Config, report.Report, error) {
 		return types.Config{}, report.Report{}, err
 	}
 
-	data, err := ioutil.ReadFile(firmwareConfigPath)
-	if os.IsNotExist(err) {
-		f.Logger.Info("QEMU firmware config was not found. Ignoring...")
-	} else if err != nil {
-		f.Logger.Err("couldn't read QEMU firmware config: %v", err)
-		return types.Config{}, report.Report{}, err
+	data := []byte{}
+	var errReadFile error
+	for _, path := range firmwareConfigPaths {
+		data, errReadFile = ioutil.ReadFile(path)
+		if os.IsNotExist(errReadFile) {
+			f.Logger.Info("QEMU firmware config was not found. Ignoring...")
+			continue
+		} else if errReadFile != nil {
+			f.Logger.Warning("couldn't read QEMU firmware config %v: %v", path, errReadFile)
+			continue
+		}
+		// successful read from path, so break to return.
+		break
+	}
+	if errReadFile != nil {
```

Comment:
```suggestion
	if errReadFile != nil && !os.IsNotExist(errReadFile) {
```
This is needed, otherwise it changes the behavior.